packages/design/src/SlideTabs/SlideTabs.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { props } from 'teleport/Apps/Apps.story';
+
+function SlideTabs({ tabs, name = 'slide-tab', onChange = () => {} }: props) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  useEffect(() => {
+    onChange(activeIndex);
+  }, [activeIndex]);
+
+  return (
+    <Wrapper>
+      <TabNav>
+        {tabs.map((tabName, tabIndex) => (
+          <TabLabel
+            htmlFor={`${name}-${tabName}`}
+            onClick={() => setActiveIndex(tabIndex)}
+            itemCount={tabs.length}
+            key={`${tabName}-${tabIndex}`}
+          >
+            {tabName}
+            <TabInput type="radio" name={name} id={`${name}-${tabName}`} />
+          </TabLabel>
+        ))}
+      </TabNav>
+      <TabSlider itemCount={tabs.length} activeIndex={activeIndex} />
+    </Wrapper>
+  );
+}
+
+type props = {
+  // A list of tab names that you'd like displayed in the list of tabs.
+  tabs: string[];
+  // The name you'd like to use for the form if using multiple tabs on the page.
+  // Default: "slide-tab"
+  name?: string;
+  // To be notified when the selected tab changes supply it with this fn.
+  onChange?: (selectedTab: number) => void;
```

Comment:
Why is this prop optional? Is there a scenario where this component would be used without this prop?

Reply:
I have no idea any longer why I made this optional :) I honestly can't think of a time when you'd want to use this component and not be notified of changes. I'll update this to make it required.